website/common/locales/en/npc.json

```diff
@@ -61,7 +61,7 @@
     "paymentMethods": "Purchase using",
 
     "classGear": "Class Gear",
-    "classGearText": "First: don't panic! Your old gear is in your inventory, and you're now wearing the apprentice equipment of your new class. Wearing your class's gear grants you a 50% bonus to stats. However, feel free to switch back to your old gear.",
+    "classGearText": "Congratulations on choosing a class! I've gone ahead and added your new basic weapon to your inventory. Take a look below to equip it!",
```

Comment:
A question for the other admins:

The text in `classGearText` now says almost the opposite of what it used to say. If we keep using that key, anyone using a language other than `en` will get the wrong message until the translators update it, and for some languages that could be months. What's our usual process in that situation? Should we use a new key for this new text and delete the old one?